### PR TITLE
Report metrics for Scheduler and Async Service

### PR DESCRIPTION
**Pull request by arunthirupathi**
Scheduler Service reports tasks currently running
and number of tasks in the queue

Async Service reports cumulative Wait time and
number of tasks waiting in the queue.

### CODE REVIEW

File: src/java/voldemort/common/service/SchedulerService.java [hunk 1]
```diff
@@ -229,18 +229,22 @@ public SchedulerThreadPool(int numThreads, final String schedulerName) {
                 private AtomicInteger threadCount = new AtomicInteger(0);
 
                 /**
-                 * This function is overridden in order to activate the daemon mode as well as
-                 * to give a human readable name to threads used by the {@link SchedulerService}.
-                 * 
-                 * Previously, this function would set the thread's name to the value of the passed-in
-                 * {@link Runnable}'s class name, but this is useless since it always ends up being a
```
Comment (FelixGV) [inline src/java/voldemort/common/service/SchedulerService.java]: Are the whitespace changes intentional?

Comment (arunthirupathi) [inline src/java/voldemort/common/service/SchedulerService.java]: Nope my eclipse is messing it up. Though I have only format changed line settings.

**Comment by FelixGV** [comment]
The change LGTM.

I guess your intent is for this metric to catch long wait times, not transient waits, right? Because if our reporting infrastructure pokes the metric once per minute, then threads that waited for less than a minute may slip in unnoticed. I think that's fine, as long as we're aware of it when looking at the dashboards.

File: src/java/voldemort/common/service/SchedulerService.java (hunk 1 above)
Comment (arunthirupathi) [inline src/java/voldemort/common/service/SchedulerService.java]: Fixed it again.

Comment (FelixGV) [inline src/java/voldemort/common/service/SchedulerService.java]: Thanks, LGTM (:

**Pull request merged**